commit faed498d0db78adc1eee6bab3a8480bcb7e17e6e
Author: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Date:   Thu Oct 12 06:50:40 2023 +0200

    hardening: x86: drop reference to removed config AMD_IOMMU_V2
    
    Commit 5a0b11a180a9 ("iommu/amd: Remove iommu_v2 module") removes the
    config AMD_IOMMU_V2.
    
    Remove the reference to this config in the x86 architecture-specific
    hardening config fragment as well.
    
    Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
    Reviewed-by: Vasant Hegde <vasant.hegde@amd.com>
    Link: https://lore.kernel.org/r/20231012045040.22088-1-lukas.bulwahn@gmail.com
    Signed-off-by: Kees Cook <keescook@chromium.org>

diff --git a/arch/x86/configs/hardening.config b/arch/x86/configs/hardening.config
index 19bb0c7a7669..7b497f3b7bc3 100644
--- a/arch/x86/configs/hardening.config
+++ b/arch/x86/configs/hardening.config
@@ -9,7 +9,6 @@ CONFIG_INTEL_IOMMU=y
 CONFIG_INTEL_IOMMU_DEFAULT_ON=y
 CONFIG_INTEL_IOMMU_SVM=y
 CONFIG_AMD_IOMMU=y
-CONFIG_AMD_IOMMU_V2=y
 
 # Enable CET Shadow Stack for userspace.
 CONFIG_X86_USER_SHADOW_STACK=y